src/py/aspen/covidhub_import/implementation.py

```diff
@@ -97,24 +126,85 @@ def import_project(
+                )
+            )
             .filter(Project.rr_project_id == rr_project_id)
             .one()
         )
 
-        group = (
-            session.query(Group)
-            .filter(Group.name == project.originating_lab)
-            .one_or_none()
-        )
-        if group is None:
-            # copy the project info into the "group"
-            group = Group(
-                name=project.originating_lab,
-                address=project.originating_address,
+        group, admin_group, cdph_group = (
+            get_or_make_group(
+                session,
+                project.originating_lab,
+                project.originating_address,
+                # FIXME: this needs to be updated.
+                email="fakeemail@dph.gov",
+            ),
+            get_or_make_group(
+                session,
+                "ADMIN",
+                "123 Main St.",
                 # FIXME: this needs to be updated.
                 email="FAKE_EMAIL@gmail.com",
+            ),
+            get_or_make_group(
+                session,
+                "CDPH",
+                "3701 N Freeway Blvd, Sacramento, CA 95834",
+                # FIXME: this needs to be updated.
+                email="cdph.internetadmin@cdph.ca.gov",
+            ),
+        )
+        covidhub_group_to_aspen_group = {
+            project.covidtracker_group.group: group,
+            covidhub_session.query(covidtracker.Group)
+            .filter(covidtracker.Group.name == "Admin")
+            .one(): admin_group,
+            covidhub_session.query(covidtracker.Group)
+            .filter(covidtracker.Group.name == "CDPH")
+            .one(): cdph_group,
+        }
+
+        covidhub_users: Iterable[covidtracker.UsersGroups] = covidhub_session.query(
+            covidtracker.UsersGroups
+        ).filter(
+            or_(
+                covidtracker.UsersGroups.group_id.in_(
+                    covidhub_session.query(covidtracker.Group.id).filter(
+                        covidtracker.Group.name.in_(["Admin", "CDPH"])
+                    )
+                ),
+                covidtracker.UsersGroups.group_id.in_(
+                    covidhub_session.query(covidtracker.GroupToProjects.group_id)
+                    .join(Project)
+                    .filter(Project.rr_project_id == rr_project_id)
+                ),
+            ),
+        )
+
+        # create the users!
+        for covidhub_user in covidhub_users:
+            # try to find the user in the auth0_usermap
+            auth0_user = auth0_usermap.get(covidhub_user.user_id, None)
+            if auth0_user is None:
+                continue
+
+            # try to create this user in the aspen db.
+            user = (
+                session.query(User).filter(User.email == auth0_user.email).one_or_none()
             )
-            session.add(group)
+            if user is None:
+                user = User()
+
+            user.name = auth0_user.nickname
```

Comment:
this is a nitpick, but i feel like we should rename `nickname` to `name`.

Reply:
We're looking at the auth0 nickname field, so it makes sense.  Had we populated our auth0 db correctly, we would be looking at username and not nickname.

Reply:
makes sense, keep nickname then 👍